### DISCUSSION TITLE
Search and open from anywhere commands

### DISCUSSION THREAD

**Discussion by Arbiter313** [notaapp/talk#324] [Questions]
Apologies if this has been answered before (did a quick search but couldn't find anything) but what is the difference in functionality between the 'Search' and 'Open from anywhere (Global)' commands? Trying them out they both seem to open the same dialog, but maybe that is only because I am using one space.

**Comment by astoilkov** [comment]
Hey. Yeah, they open the same dialog. The only difference is that the 'Open from anywhere (Global)' dialog can be opened when pressing the shortcut and the app isn't on focus.

Your question makes me think that we should change the name of that command to reduce confusion.

Thanks for the feedback!
